Integrations/integration-Twilio.yml

```diff
@@ -71,7 +71,24 @@ script:
     switch (command) {
         // This is the call made when pressing the integration test button.
         case 'test-module':
-            sendRequest(encodeToURLQuery(params.from,params.from,'Test'));
+            // from Twilio instructions, in order to ger the list of accounts, the below curl command should be used.
+            // using this as a method to test tht the credentails and connectivity are good, without using sms buckets
+            // curl -G https://api.twilio.com/2010-04-01/Accounts -u '[YOUR ACCOUNT SID]:[YOUR AUTH TOKEN]'
+
+            var res = http(
```

Comment:
instead duplicating `sendRequest` - just change `sendRequest` to get an url argument, then pass `params.server.replace(/[\/]+$/, '')` in the test and `params.server.replace(/[\/]+$/, '')+'/'+params.sid+'/Messages.json'` in other calls